cardboardlint/common.py

```diff
@@ -70,13 +73,13 @@ def indiff(self, files_lines):
         ----------
         files_lines : dict
             Dictionary of filename to the set of line numbers (that have been modified)
-            Result of git diff from run_diff function
+            Result of git diff from run_diff function. The set of line numbers may also
+            be None, indicating that all lines should be considered in that file.
         """
-        if files_lines is not None:
-            line_numbers = files_lines.get(self.filename)
-            return line_numbers is not None and self.lineno in line_numbers
-        else:
-            return True
+        if self.filename in files_lines:
+            line_numbers = files_lines[self.filename]
+            return line_numbers is None or self.lineno is None or self.lineno in line_numbers
+        return False
```

Comment:
I had that originally and PyLint complained. I'm going to disable the warning and fix it back.